src/common/snippets/include/snippets/op/blockedparameter.hpp

```diff
@@ -19,6 +19,7 @@ namespace op {
 class BlockedParameter : public ngraph::op::Parameter {
 public:
     OPENVINO_OP("BlockedParameter", "SnippetsOpset", ngraph::op::Parameter);
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
why do we need it?
CC @ilyachur

Reply:
Base class has BWDCMP_RTTI_DECLARATION. So all childs should also have BWDCMP_RTTI_DECLARATION, otherwise it will lead to errors.